src/routes/activities/+page.svelte

```diff
@@ -37,36 +108,15 @@
 				{/each}
 			{/if}
 		</div>
-		<InfiniteLoading
-			forceUseInfiniteWrapper
-			on:infinite={async ({ detail }) => {
-				if (activities.length < 6) return;
-
-				activitiesLoading = true;
-
-				newActivities = await trpc($page).activity.getActivities.query({
-					take: 6,
-					cursor,
-					search: data.query
-				});
-
-				if (newActivities.length === 0) {
-					detail.complete();
-					activitiesLoading = false;
-					return;
-				}
-
-				activities = [...activities, ...newActivities];
-				cursor = newActivities[newActivities.length - 1].paginationId;
-
-				detail.loaded();
-				activitiesLoading = false;
-			}}
-		>
-			<div slot="spinner" />
-			<div slot="noResults" />
-			<div slot="noMore" />
-		</InfiniteLoading>
+		{#if !hasMore}
+			<div class="flex justify-center items-center flex-col mt-12">
+				<h1 data-testid="error-fallback-title" class="text-xl font-medium text-center">
+					{$_('a-default.no-more-results')}
+				</h1>
+			</div>
+		{:else}
+			<div use:inview={{}} on:change={handleChange} />
```

Comment:
Can we write like this?
```suggestion
			<div use:inview on:change={handleChange} />
```